expression/builtin.go

```diff
@@ -799,6 +801,7 @@ var funcs = map[string]functionClass{
 	ast.UUIDToBin:       &uuidToBinFunctionClass{baseFunctionClass{ast.UUIDToBin, 1, 2}},
 	ast.BinToUUID:       &binToUUIDFunctionClass{baseFunctionClass{ast.BinToUUID, 1, 2}},
 	ast.TiDBShard:       &tidbShardFunctionClass{baseFunctionClass{ast.TiDBShard, 1, 1}},
+	groupingImplName:    &groupingImplFunctionClass{baseFunctionClass{groupingImplName, 1, 1}},
```

Comment:
Is this change still needed for this pr?

Reply:
> Is this change still needed for this pr?

I will delete it